example/autoencoder/model.py

```diff
@@ -53,7 +54,7 @@ def extract_feature(sym, args, auxs, data_iter, N, xpu=mx.cpu()):
 
 
 class MXModel(object):
-    def __init__(self, xpu=mx.cpu(), *args, **kwargs):
+    def __init__(self, *args, xpu=mx.cpu(), **kwargs):
```

Comment:
why?

Reply:
I forget that why I change it, and it seems like the position of *args does not impact anything in the module, therefore I intend to keep the original design(*args parameter at 2nd).